web-admin/src/routes/-/[organization]/[project]/dashboard/[name]/+page.svelte

```diff
@@ -1,9 +1,20 @@
 <script lang="ts">
   import { page } from "$app/stores";
   import { Dashboard } from "@rilldata/web-common/features/dashboards";
+  import { useDashboardStore } from "@rilldata/web-common/features/dashboards/dashboard-stores";
+  import { StateSyncManager } from "@rilldata/web-common/features/dashboards/proto-state/StateSyncManager";
 
   const metricViewName: string = $page.params.name;
-  $: console.log("metricViewName", metricViewName);
+  const stateSyncManager = new StateSyncManager(metricViewName);
+
+  $: metricsExplorer = useDashboardStore(metricViewName);
```

Comment:
Now for consistency, I think it makes sense to switch this from `metricsExplorer` to `dashboardStore`

Reply:
honestly there is too much inconsistancy in the naming in dashboard feature. It is called `metricsExplorer` in all components. So lets refactor that in one shot.

Reply:
Agreed we're inconsistent right now. I was thinking new code could use the "dashboard" terminology and over time we'd transition to the right state.